webpack/webpack4-module-minifier-plugin/package.json

```diff
@@ -44,11 +44,12 @@
   },
   "devDependencies": {
     "@rushstack/heft": "workspace:*",
-    "local-node-rig": "workspace:*",
-    "@types/webpack": "4.41.32",
+    "@types/node": "20.17.19",
     "@types/webpack-sources": "1.4.2",
-    "webpack": "~4.47.0",
-    "webpack-sources": "~1.4.3"
+    "@types/webpack": "4.41.32",
+    "local-node-rig": "workspace:*",
+    "webpack-sources": "~1.4.3",
+    "webpack": "~4.47.0"
```

Comment:
Just double-checking, do we have any feature in Rush to keep these organized? / Should we just enable the prettier rule for the repo? See: https://github.com/microsoft/rushstack/issues/2496

Reply:
We currently don't have a Rush feature. We've had Rush both sort and not sort in the past, and people have taken issue with it both ways. I'm down to pull the sorting prettier tool in.